apps/bare-expo/android/app/src/main/AndroidManifest.xml

```diff
@@ -9,6 +9,7 @@
   <uses-permission android:name="android.permission.MODIFY_AUDIO_SETTINGS"/>
   <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION"/>
   <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
+  <uses-permission android:name="android.permission.ACCESS_BACKGROUND_LOCATION"/>
```

Comment:
is that intentional to add the permission?

Reply:
Yes, it was. It's not related, but during testing, I noticed that this permission is missing in the bare expo app.